<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccs::main::TermCriteriaConfigurationWidget</class>
 <widget class="QGroupBox" name="ccs::main::TermCriteriaConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>192</height>
   </rect>
  </property>
  <property name="title">
   <string>Termination Criteria</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QRadioButton" name="epsilonRadioButton">
     <property name="text">
      <string>Epsi&amp;lon only:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="epsilonLineEdit">
     <property name="placeholderText">
      <string>DBL_EPSILON</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="maxIterationsRadioButton">
     <property name="text">
      <string>Ma&amp;x. iterations only:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="maxIterationsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="bothRadioButton">
     <property name="text">
      <string>Epsilon and max. i&amp;terations</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
